src/main.ts

```diff
@@ -49,6 +49,15 @@ app.config.globalProperties.$filters = {
     // TODO Fix this setDefault should set the default timezone instead of getting it everytiem and setting the tz
     return moment.utc(value, inFormat).tz(userProfile.userTimeZone).format(outFormat ? outFormat : 'MM-DD-YYYY');
   },
+  getOrderIdentificationId(identifications: any, id: string) {
```

Comment:
If we have not used this utility anywhere, then why we are implementing it in this PR?